main/webapp/modules/core/langs/translation-fr.json

```diff
@@ -13,6 +13,7 @@
     "core-index/version": "Version",
     "core-index/error-rename": "Échec du renommage du projet :",
     "core-index/pref-key": "Valeur de la préférence :",
+    "core-index/prefs-loading-failed": "La lecture des préférences n’a pas réussie.",
```

Comment:
should be "n'a pas réussi"

Reply:
Oh, merde, un accord de participe passé avec le verbe avoir… Toujours un peu traitre… ;-). A.